actor/src/main/scala/org/apache/pekko/dispatch/ThreadPoolBuilder.scala

```diff
@@ -235,6 +235,18 @@ final case class MonitorableThreadFactory(
   }
 }
 
+class MonitorableCarrierThreadFactory(name: String)
+    extends ForkJoinPool.ForkJoinWorkerThreadFactory {
+  private val counter = new AtomicLong(0L)
+
+  def newThread(pool: ForkJoinPool): ForkJoinWorkerThread = {
+    val thread = VirtualThreadSupport.CarrieThreadFactory.newThread(pool)
+    // Name of the threads for the ForkJoinPool are not customizable. Change it here.
+    thread.setName(name + "-" + "CarrierThread" + "-" + counter.incrementAndGet())
+    thread
```

Comment:
it's a `CarrierThread` nothing needs to be set, only the name.

Reply:
CarrieThreadFactory looks like a typo - maybe should be CarrierThreadFactory

Reply:
Yes, thanks, will update.